.prettierignore

```diff
@@ -0,0 +1,20 @@
+# Public Root
+public/
+
+# Dependencies
+node_modules/
+themes/
+vendor/
+
+# Files managed by other tools
+composer.json
+composer.lock
+package-lock.json
+package.json
+
+# Bitbucket Pipeline
+bitbucket-pipelines.yml
+docker-compose.yml
+
+# Ignore YML files
+*.yml
```

Comment:
We want to format these with prettier - most of these should be fine to format with `prettier`.

Reply:
I found that prettier messes with the readability of YML files and in the case of bitbucket, it potentially borks it. As for lock files. Lock files, they would get rewritten either way i understand. ./public will be generated assets. Let me know which ones should not be here please.

Reply:
The whole point of `prettier` is to improve readability, so I'd expect changes its making are to help improve general readability :)

Usually this is just a case of making sure you're using the right [YAML multiline syntax](https://yaml-multiline.info/).
The dense content of the pipeline files is actually why I'd like it to not be ignored: if we're not confident our automatic formatting tools won't bork a thing, we should try to improve the thing (i.e you can extract stuff into their own scripts - I did that for one project, putting things in `ci-scripts/`).

The lockfiles don't have to be ignored because while they should be consistent, they can be manually edited - so not ignoring them means prettier can correct if needed.

Reply:
left some stuff in the pipelines without autoformatting as they are due to be refactored into bin/ci scripts instead of being ran directly in the pipeline.